lib/typhoeus/request.rb

```diff
@@ -214,7 +214,9 @@ def set_defaults
       options[:headers] = {'User-Agent' => default_user_agent}.merge(options[:headers] || {})
       options[:verbose] = Typhoeus::Config.verbose if options[:verbose].nil? && !Typhoeus::Config.verbose.nil?
       options[:maxredirs] ||= 50
+      @options[:headers]['Expect'] ||= ''
```

Comment:
1. What a great fix for such annoying issue! kudos @AvnerCohen 
2. Why not using just ```options``` (without @)?
3. Why do we need the conditional here? don't we want to always override it? if so, it can be done together with the previous ```headers``` assignment (https://github.com/AvnerCohen/typhoeus/blob/d93c976859d0fa646a9e732f13777cedb6dd7811/lib/typhoeus/request.rb#L214)
```Ruby
options[:headers] = {'User-Agent' => default_user_agent, 'Expect' => ''}.merge(options[:headers] || {})
```